# Effector name
string[] effector

# Effector position
float64[] x
float64[] y
float64[] z

# Angles
float64[] roll
float64[] pitch
float64[] yaw

# Verify if there is one raised foot
string support_polygon
